internal/handlers/common/listindexes.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+
+	"go.uber.org/zap"
+
+	"github.com/FerretDB/FerretDB/internal/handlers/commonerrors"
+	"github.com/FerretDB/FerretDB/internal/types"
+)
+
+// ListIndexesParams contains `listIndexes` command parameters.
+type ListIndexesParams struct {
+	DB         string
+	Collection string
+}
+
+// GetListIndexesParams returns `listIndexes` command parameters.
+func GetListIndexesParams(document *types.Document, l *zap.Logger) (*ListIndexesParams, error) {
```

Comment:
Let's remove that: all index-related methods do not receives tens of parameters like `find`

Reply:
Well, 24 lines of code that are absolutely the same for all the current and future handlers. But if it matters, I can copy-paste, yes.

Reply:
For better or worse, we currently do that only for commands with a lot of parameters. Doing that there while keeping, for example, `insert` separate would make our code base less consistent

Reply:
That would be nice to know how much "a lot" is. So, four (the current case) is not a lot. Is five a lot?

Reply:
That's the best answer I can provide: https://www.youtube.com/watch?v=iG2YxzjC9WI 😆 

Seriously, though, that's not about the number itself, but about consistency